README.md

```diff
@@ -70,132 +68,167 @@
+- On the terminal run this command -
+
+```sh
+ git clone [https://github.com/tsohleDev/To-do-planner.git]
+```
+
+- Go to the repository folder using command prompt
+```sh
+ cd To-do-planner
+```
 
-To get a local copy up and running, follow these steps.
+- Install the dev dependencies for linters run
+```sh
+ npm install.
+```
+- To start run
+```sh
+ npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ### Prerequisites
 
 In order to run this project you need:
 
-- Have visual studio Code in your PC.
-- Have a verified GitHub account.
-- Have a stable browser version.
+-A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
 
-### Setup
+-VSCode or any other equivalent code editor
 
-- Clone this repository to your desired folder using terminal:
-  git clone https://github.com/ReemMohamedAbdelfatah/toDOLISTapp.git
+-Node Package Manager (For running, debuging, or installing dependancies to be used)
 
-### Install
+node and npm:
 
-Install this project with:
+-Linux
+```sh
+  sudo apt install nodejs npm
+```
+-windows 
 
-- webpack and linters.
+[download and install](https://nodejs.org/en/)
 
-### Usage
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-To run the project, execute the following command:
+### Setup
 
-- npm run build
-- npm run dev
+Clone this repository to your desired folder:
 
-### Run tests
+- Open the command prompt in the created directory.
 
-To run tests, run the following command:
+- On the terminal run this command 
+```sh
+git clone https://github.com/tsohleDev/To-do-planner.git
+```
 
-Coming Soon ....
+- Go to the repository folder using command prompt
+```sh
+ cd To-do-planner
+```
 
-### Deployment
+- Install the dev dependencies for linters 
+```sh
+ run npm install.
+```
 
-You can deploy this project using:
+- To start run 
+```sh
+npm start
+```
 
-Coming Soon ....
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 👥 Authors <a name="authors"></a>
+<!-- Author -->
+
+## 👤**Authors** <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
+<h4>Reem Muhammed </h4>
 
-👤 **Reem Muhammed**
+<img src="/images/github.svg" alt="logo" width="18"  height="18" />[githubhandle](https://github.com/ReemMohamedAbdelfatah)
+<br>
+<img src="/images/linkedin.svg" alt="logo" width="18"  height="18" />[linkedIn](https://www.linkedin.com/in/reem-abd-el-fatah-a07543116)
 
-- GitHub: [@github](https://github.com/ReemMohamedAbdelfatah)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/reem-abd-el-fatah-a07543116)
+<h4>Tsohle Mokhemisi </h4>
+
+<img src="/images/github.svg" alt="logo" width="18"  height="18" />[githubhandle](https://github.com/tsohleDev2)
+<br>
+<img src="/images/twitter.svg" alt="logo" width="18"  height="18" /> [twitterhandle](https://twitter.com/RealTsohle)
+<br>
+<img src="/images/linkedin.svg" alt="logo" width="18"  height="18" />[linkedIn](https://www.linkedin.com/in/tsohle-mokhemisi-3687401b2/)
```

Comment:
- [ ] These images are all broken and this gives your README an unprofessional look. Kindly fix it.